doc/cli.markdown

```diff
@@ -1,83 +1,74 @@
 # Balena CLI Documentation
```

Comment:
This is the the markdown that constitutes the [CLI docs web page](https://www.balena.io/docs/reference/cli/) in full. This file is auto generated out of README.md and capitanodoc.ts. Click the `'View file'` button (just above, to the right) to see this page nicely rendered.